apps/explorer/src/components/orders/OrderDetails/context/FillsTableContext.tsx

```diff
@@ -5,7 +5,7 @@ import { Trade } from 'api/operator'
 import { TableState, TableStateSetters } from '../../../../explorer/components/TokensTableWidget/useTable'
 
 type CommonState = {
-  trades: Trade[]
+  data: Trade[]
   isLoading: boolean
```

Comment:
@shoom3301 this one was hard to debug, cause the error comes from some unsafety casting done in the pagination.

I could see the context being injected:
<img width="1528" alt="Screenshot at Dec 18 12-30-13" src="https://github.com/user-attachments/assets/a88cc8a0-eb0f-4870-86cc-51d39d939340" />


However, because there's this type casting here:
<img width="758" alt="image" src="https://github.com/user-attachments/assets/da56632b-3a6d-4aad-be42-4f07756f127f" />


We don't get compile errors. Basically, was assuming there will be something called data, but instead injected something called `trades`

This PR is about fixing the issue to get it out, but maybe we should add to the technical debt this too

Reply:
I think the problem comes from [here](https://github.com/cowprotocol/cowswap/blob/09ba94235846b24e5ae86deb28b3db616cd86aa7/apps/explorer/src/explorer/components/common/TablePagination/index.tsx#L125) because the context is casted as any here 

<img width="907" alt="image" src="https://github.com/user-attachments/assets/89d1ff2e-987d-44c9-b41a-72f5f28155fa" />

Reply:
Got it, thanks for the information.
This context should definetely be refactored to jotai atom, at least in terms of code-base consistency.